README.md

```diff
@@ -1 +1,301 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+Testing my skills on LInters, Gitflow and Pull Request
+
+**[Hello Microverse]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://stylelint.io/">Linters</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[It has a homepage]**
+- **[Header tag has color blue]**
+- **[SEO Friendly]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!-General requirements
+Once you finish your project, double-check to make sure your repo is ready:
+
+Make sure that there are no linter errors.
+Make sure that you used correct (GitHub flow).
+Make sure that you documented your work.
+Requirements
+Remember to document your work in a professional way.
+Create a new GitHub repo.
+Create a new branch according to the GitHub flow rules.
+Add a .gitignore file.
+Add node_modules to gitignore file.
+Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
+Add descriptive README file to your project - please use this template but remember to customize it to your project.
+Add 3 files (each one in a separate commit):
+A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
+A .css file that includes one class with styles for your header (make it in your favorite color).
+A test.md file with the text "This file should be ignored by git".
+Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
+Open a pull request.-
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
```

Comment:
- [ ] kindly edit this so that it'S related to your project to make your Readme more descriptive and not confuse the people who will be reviewing your project